package.json

```diff
@@ -74,10 +75,10 @@
     "react-test-renderer": "^16.12.0",
     "semantic-release": "^17.0.0",
     "svgo": "^1.3.2",
-    "tailwindcss": "^1.9.6",
-    "ts-jest": "^25.0.0",
+    "tailwindcss": "^2.1.2",
+    "ts-jest": "25.0.0",
     "ts-loader": "^9.0.0",
-    "typescript": "^4.0.0"
+    "typescript": "4.0"
```

Comment:
I would not care about pinning versions of dev dependencies too much. what's your reasoning for this change?